docs/_docs/developer-guide/scheduler-stress-testing.md

```
@@ -0,0 +1,204 @@
+parent: Reference
+layout: default
+linkTitle: "Scheduler Stress Testing"
+date: 2026-06-12
+description: >
+  How to run, tune, and interpret the Rust scheduler's booking and accounting
+  stress suite, locally and in CI
+---
+
+# Scheduler Stress Testing
+
+### Running and interpreting the booking + accounting stress suite
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)